lib/payrix/resource/txns.rb

```diff
@@ -26,7 +26,7 @@ def initialize(params)
                :authentication, :authenticationId, :cofType, :copyReason, :originalApproved, 
                :currencyConversion, :serviceCode, :authTokenCustomer, :debtRepayment, :statement, 
                :convenienceFee, :surcharge, :channel, :funded, :fundingEnabled, 
-               :requestSequence, :processedSequence]
+               :requestSequence, :processedSequence, :txnResults, :payment]
```

Comment:
A resource attribute has to be defined on the inherited resource in order to access it through the `expand` parameter. In Payrix, the `txnResults` resource gives information about errors, and `payment` gives information about the payment method used.